src/OSPSuite.UI/Services/CsvSeparatorSelector.cs

```diff
@@ -13,13 +13,13 @@ public CsvSeparatorSelector(IApplicationController applicationController)
          _applicationController = applicationController;
       }
 
-      public CSVSeparators GetCsvSeparator(string fileName)
+      public (char? ColumnSeparator, char DecimalSeparator) GetCsvSeparator(string fileName)
       {
          using (var csvSeparatorPresenter = _applicationController.Start<ICSVSeparatorSelectorPresenter>())
          {
             csvSeparatorPresenter.SetFileName(fileName);
 
-            return csvSeparatorPresenter.Canceled() ? null : csvSeparatorPresenter.GetCSVSeparators();
+            return csvSeparatorPresenter.Canceled() ? (null, '.') : csvSeparatorPresenter.GetCSVSeparators();
```

Comment:
This should come from the presenter returning . Shouldn't have to ask cancel or not. Tell don't ask